cmd/backups_new.go

```diff
@@ -54,6 +55,16 @@ backup.`,
 			command = "new/partial"
 		}
 
+		compressed, err := cmd.Flags().GetBool("compressed")
+		if err == nil && cmd.Flags().Changed("compressed") {
+			options["compressed"] = compressed
+		}
+
+		uncompressed, err := cmd.Flags().GetBool("uncompressed")
+		if err == nil && cmd.Flags().Changed("uncompressed") {
+			options["compressed"] = !uncompressed
+		}
```

Comment:
Looking into supervisor options for see handling of flags options

Reply:
Ok, just followed what was done for other flags in the code around this, will check better!

Reply:
Thanks for handling this!